libs/components/src/next/form/datepicker/stories/vl-datepicker.stories-util.ts

```diff
@@ -0,0 +1,19 @@
+import flatpickr from 'flatpickr';
+
+export const formatEpoch = (epoch: number | string | symbol, format: string | symbol) => {
+    if (epoch && (typeof epoch === 'string' || typeof epoch === 'number')) {
+        // @ts-ignore
+        const dateFormat = format && typeof format !== 'symbol' ? format : 'd.m.Y';
+        return flatpickr.formatDate(new Date(Number(epoch)), dateFormat);
+    } else {
+        return undefined;
+    }
+};
+
+export const formatTime = (time: string | symbol) => {
+    if (time && typeof time === 'string') {
+        return flatpickr.formatDate(new Date(`1970-01-01T${time}`), 'H:i');
```

Comment:
Is er een betere manier om dit te doen ipv een fake datum te gebruiken?

Reply:
ik bekijk het